gradoop-store/gradoop-hbase/src/main/java/org/gradoop/storage/impl/hbase/handler/HBaseEdgeHandler.java

```diff
@@ -35,16 +35,29 @@
 
 /**
  * Used to read/write EPGM edge data from/to a HBase table.
- * <p>
- * Edge data in HBase:
- * <p>
- * |---------|---------------------------------------------|-------|
- * | row-key | meta                                        | data  |
- * |---------|----------|------------|------------|--------|-------|
- * | "0"     | label    | source     | target     | graphs | since |
- * |         |----------|------------|------------|--------|-------|
- * |         | "knows"  | <Person.0> | <Person.1> | [0,1]  | 2014  |
- * |---------|----------|------------|------------|--------|-------|
+ * <table>
```

Comment:
Apparently you can use `<table border="1">` here to get a border around the table. Please try that